ui/desktop/src/components/SplashPills.tsx

```diff
@@ -5,11 +5,8 @@ function SplashPill({ content, append, className = '', longForm = '' }) {
     <div
       className={`px-4 py-2 text-sm text-center text-textSubtle dark:text-textStandard cursor-pointer border border-borderSubtle hover:bg-bgSubtle rounded-full transition-all duration-150 ${className}`}
       onClick={async () => {
-        const message = {
-          content: longForm || content,
-          role: 'user',
```

Comment:
It doesn't look to me like the new `append` implementation does anything to explicitly set that the message being appended is from the user https://github.com/block/goose/pull/1393/files#diff-b68456893042db8d6d236bc76aef8598d4ccabb69fb65a08158f2381d25492a2R336

Is there something I am missing that makes this work?

Reply:
yes later we define append as text => createUserMessage(text)